packages/vue-pdf-viewer/src/components/PDFViewer.vue

```diff
@@ -227,6 +245,9 @@ export default {
 
                 const event = new Event('PDFViewer:pagesLoaded')
                 window.dispatchEvent(event)
+
+                const scrollbarWidth = this.$refs.viewerContainer.offsetWidth - this.$refs.viewerContainer.clientWidth
+                this.$refs.viewerControls.style.width = `calc(100% - ${scrollbarWidth}px)`
```

Comment:
The controls were not aligned correctly because the viewer-container has a scrollbar most of the time and the file is centered based on width without scrollbar but the controls don't. So the controls were to wide. Do you have a better idea to solve it?

Reply:
Is the goal to always have the controls only over the actual pages and not the viewer?

Reply:
I'm not sure what you mean here. See the red marks in the screenshot. On the right side it would look weird since the scrollbar is within the width.
![image](https://user-images.githubusercontent.com/397457/221839445-1fc0c78e-ee20-44e2-a229-8578950878eb.png)

Reply:
Ah I only looked at the screenshot you attached to the PR description and there it looks like the controls could also be aligned to the PDF page and not the viewport. Makes sense like this.